### PR TITLE
JAX-WS + XML WS: Add Test for Colocated policy-attachment-server/client.xml files

### PR DESCRIPTION
**Pull request by neuwerk**
This pull request handles the following:

1.  It adds the `ColocatedPolicyAttachmentTest` for testing colocated policy-attachment-server/client.xml files that attach WS-Addressing Policy to the endpoints
2.  It fixes various `NoSuchMethodExceptions` in CXF's `SoapMessage` implementation by recompiling the classes that use it, in order to address a parent class being recompiled with override changes. 
3.  It pushes more CXF trace down into the debug level to prevent them from going to messages.log

### CODE REVIEW

File: dev/com.ibm.ws.jaxws.2.2.webcontainer_fat_extended/test-applications/policyAttachmentsClient1/src/com/ibm/ws/policyattachments/client1/service1/HelloClientReplyToHandler.java [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+import javax.xml.soap.SOAPElement;
+import javax.xml.soap.SOAPException;
+import javax.xml.soap.SOAPHeader;
+import javax.xml.soap.SOAPMessage;
+import javax.xml.ws.handler.MessageContext;
+import javax.xml.ws.handler.soap.SOAPHandler;
+import javax.xml.ws.handler.soap.SOAPMessageContext;
+
+/**
+ * The purpose of this SOAPHandler is to modify the ReplyTo SOAPHeaders default Address value from
```
Comment (sawadood) [inline dev/com.ibm.ws.jaxws.2.2.webcontainer_fat_extended/test-applications/policyAttachmentsClient1/src/com/ibm/ws/policyattachments/client1/service1/HelloClientReplyToHandler.java:27]: default Address value from to, from address is missing

File: dev/com.ibm.ws.jaxws.2.2.webcontainer_fat_extended/test-applications/policyAttachmentsClient1/src/com/ibm/ws/policyattachments/client1/service1/HelloService3.java [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+import javax.xml.ws.WebServiceFeature;
+
+/**
+ * This class was generated by the JAX-WS RI.
+ * JAX-WS RI 2.2.9-b130926.1035
+ * Generated source version: 2.2
+ *
+ */
+@WebServiceClient(name = "HelloService3", targetNamespace = "http://service1.policyattachments.ws.ibm.com/",
+                  wsdlLocation = "http://localhost:8010/policyAttachmentsClient1/HelloService3?wsdl")
```
Comment (sawadood) [inline dev/com.ibm.ws.jaxws.2.2.webcontainer_fat_extended/test-applications/policyAttachmentsClient1/src/com/ibm/ws/policyattachments/client1/service1/HelloService3.java:22]: Why are we using hard-coded port # 8010 here?

File: dev/com.ibm.ws.jaxws.2.2.webcontainer_fat_extended/test-applications/policyAttachmentsClient1/src/com/ibm/ws/policyattachments/client1/service1/HelloService4.java [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+import javax.xml.ws.WebServiceFeature;
+
+/**
+ * This class was generated by the JAX-WS RI.
+ * JAX-WS RI 2.2.9-b130926.1035
+ * Generated source version: 2.2
+ *
+ */
+@WebServiceClient(name = "HelloService4", targetNamespace = "http://service1.policyattachments.ws.ibm.com/",
+                  wsdlLocation = "http://localhost:8010/policyAttachmentsClient1/HelloService4?wsdl")
```
Comment (sawadood) [inline dev/com.ibm.ws.jaxws.2.2.webcontainer_fat_extended/test-applications/policyAttachmentsClient1/src/com/ibm/ws/policyattachments/client1/service1/HelloService4.java:21]: hard-coded port # 8010?

**Review by sawadood — APPROVED** [review]

File: dev/com.ibm.ws.jaxws.2.2.webcontainer_fat_extended/test-applications/policyAttachmentsClient1/src/com/ibm/ws/policyattachments/client1/service1/HelloService4.java (hunk 3 above)
Comment (neuwerk) [reply dev/com.ibm.ws.jaxws.2.2.webcontainer_fat_extended/test-applications/policyAttachmentsClient1/src/com/ibm/ws/policyattachments/client1/service1/HelloService4.java:21]: See the change in `dev/com.ibm.ws.jaxws.2.2.webcontainer_fat_extended/publish/servers/ColocatedDynamicPolicyAttachments/server.xml` where I addressed this issue.

File: dev/com.ibm.ws.jaxws.2.2.webcontainer_fat_extended/test-applications/policyAttachmentsClient1/src/com/ibm/ws/policyattachments/client1/service1/HelloService3.java (hunk 2 above)
Comment (neuwerk) [reply dev/com.ibm.ws.jaxws.2.2.webcontainer_fat_extended/test-applications/policyAttachmentsClient1/src/com/ibm/ws/policyattachments/client1/service1/HelloService3.java:22]: See the change in `dev/com.ibm.ws.jaxws.2.2.webcontainer_fat_extended/publish/servers/ColocatedDynamicPolicyAttachments/server.xml` where I addressed this issue.

**Comment by neuwerk** [comment]
#build

**Comment by neuwerk** [comment]
#build

**Comment by neuwerk** [comment]
#build

**Comment by neuwerk** [comment]
#run-libby-bot

**Pull request merged**